lib/ansible/modules/add_host.py

```diff
@@ -30,26 +30,21 @@
     type: list
     aliases: [ group, groupname ]
 extends_documentation_fragment:
-- action_common_attributes
+  - action_common_attributes
+  - action_core
 attributes:
     action:
         support: full
-    async:
-        support: none
-    become:
-        support: none
+    core:
+      description: While parts of this action are implemented in core, other parts are still available as normal plugins and can be partially overridden
```

Comment:
Shouldn't this be `details`...

Reply:
no, this describes the 'core' attribute and is the place were it is defined, details should be done by the importers